cli/cmd/deploy.go

```diff
@@ -0,0 +1,53 @@
+package cmd
```

Comment:
create `Deploy` package instead of having it in CMD 😄

Reply:
Actually I did this on purpose, because I was wondering if it was better. Essentially the `cmd` package should only contain entrypoints anyway, and no real logic, so it is a bit overkill to have some many packages with very little logic in it.

But I can change it, to be more consistent, and we can consider if we should refactor in the future :)

Reply:
Changed it :)